### ISSUE TITLE
[4.7.0] Property 'observable' does not exist on type 'SymbolConstructor'.

### ISSUE THREAD

**Issue by wKovacs64** [davidkpiano/xstate#835]
**Description**

TypeScript is throwing an error as of `xstate@4.7.0`.

**Expected Result**

No errors.

**Actual Result**

```
node_modules/xstate/lib/interpreter.d.ts(171,13): Error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.
```

**Reproduction**

- [CodeSandbox template](https://codesandbox.io/s/xstate-example-template-m4ckv)

The default reproduction example includes the problem, but you have to download it from CodeSandbox and run `tsc` locally to see it, as far as I know.

**Additional context**

xstate v4.6.7 is OK, but v4.7.0 and v4.7.1 (at the time of this writing) exhibit the error.

**Comment by davidkpiano** [comment]
When building locally, if you change this in `packages/core/src/interpreter.ts:1216`, does it fix the problem?

```diff
- public [Symbol.observable]() {
+ public [(Symbol as any).observable]() {
    return this;
  }
```

**Comment by wKovacs64** [comment]
Yes, it appears so. Rebuilding after making that change completely removes the offending line from the built types (`packages/core/lib/interpreter.d.ts:171`):

```diff
    toJSON(): {
        id: string;
    };
-   [Symbol.observable](): this;
}
export declare function spawn<TC, TE extends EventObject>(entity: StateMachine<TC, any, TE>, nameOrOptions?: string | SpawnOptions): Interpreter<TC, any, TE>;
export declare function spawn(entity: Spawnable, nameOrOptions?: string | SpawnOptions): Actor;
```

**Comment by davidkpiano** [comment]
That's fine; that line is for run-time compatibility with TC39 observables anyway.

**Comment by qtiki** [comment]
> That's fine; that line is for run-time compatibility with TC39 observables anyway.

I don't think this is fine. Creating an empty TypeScript project with xstate doesn't compile so basically the whole package in NPM is broken for TypeScript users:

package.json:
```json
{
  "name": "xstate-test",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "xstate": "^4.7.1"
  },
  "devDependencies": {
    "typescript": "^3.7.2"
  }
}
```

tsconfig.json:
```json
{
  "compilerOptions": {
    "lib": [
      "dom",
      "es2015"
    ]
  },
  "include": [
    "test.ts"
  ]
}
```

test.ts:
```ts
import * as xstate from 'xstate';
```

Trying to compile this project with `tsc` gives the error mentioned in the issue title.

**Comment by wKovacs64** [comment]
@qtiki The fix hasn't made it into a published version, yet.

**Comment by qtiki** [comment]
Oh I thought you were discussing a local workaround and not a fix to the package. I'll revert to an older version then while waiting for the fix. Thanks.

**Comment by wKovacs64** [comment]
You can patch it yourself with [patch-package](https://www.npmjs.com/package/patch-package) if you want to use the latest version until the fix is released (that's what I'm doing). Or revert, yeah.

**Comment by davidkpiano** [comment]
Published as 4.7.2 @qtiki

**Comment by qtiki** [comment]
> Published as 4.7.2 @qtiki

Can confirm it's working now. 👍
